### DISCUSSION TITLE
切换全屏后无法加载视频画面

### DISCUSSION THREAD

**Discussion by St-Cai** [Richasy/Bili.Uwp#212] [问答]
在默认窗口/全屏/小窗/全窗口模式间切换的时候视频画面一片白，但在继续播放有声音，切换清晰度后可以恢复正常。旧版也会出现一样的问题，这是bug吗？

**Comment by Richasy** [comment]
算，但可能不是应用的问题，Issue或Discussions里有类似的问题，但基本都是显卡设置的原因

**Reply by St-Cai** [reply]
求解要如何设置，笔记本显卡是AMD核显和3060，系统是win10 21H1
